infra/terraform/modules/data_access/github_webhooks_handler/tests/conftest.py

```diff
@@ -0,0 +1,140 @@
+from hashlib import sha1
+from moto import mock_sns
+import boto3
+import hmac
+import json
+import os
+
+import pytest
+
+
+#################
+#  Tests state  #
+#################
```

Comment:
I knew you were going to say something about these. I guess I wanted to keep things tidy, but good point about "another thing to maintain" so I'll remove them 👍